app/policies/portfolio_item_policy.rb

```diff
@@ -39,6 +39,9 @@ def set_approval?
       rbac_access.approval_workflow_check
   end
 
+  alias tag? set_approval?
```

Comment:
I have a question here: the permission for `tag?` and `untag?` in this way will be the same as `set_approval?`, which totally depends on actions on `workflow`. But this `tag?`, `untag?` also uses for tags on `OrderProcess` based on following `TaggingMixin`, right? They share the same permission check, is this what we want?

Reply:
Right, I think that `tag?` and `untag?` for portfolio items should have the same permission check as `set_approval?`. You need to be able to update the portfolio, and you need to be able to read, link/unlink approval workflows. If we wanted separate "tag/untag" permissions, we would have to update the rbac-config.

For order process it goes through the `order_process_policy`, which only ensures that you have the link/unlink permissions.

Reply:
@eclarizio Would it make sense to add the tag and untag permissions into the rbac config? That might be a cleaner way of implementing this. We can use it for OrderProcess to start with and then later on add it to the SetApproval action for Portfolio and PortfolioItems.

Reply:
@mkanoor Yeah, we could do that. I think that's probably outside the scope of this PR though, and it's easy to do since all we have to do after the rbac-config is updated is to adjust the logic in the policies to no longer use aliases.